components/Tags/Tags.styles.ts

```diff
@@ -8,42 +8,27 @@ import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
 export const tagsStyles = makeStyles((theme: Theme) =>
   createStyles({
     root: {
-      marginTop: '1rem',
-      marginBottom: '1rem',
+      display: 'flex',
+      alignItems: 'center',
+      flexWrap: 'wrap',
+      margin: '.5rem 0',
       color: theme.palette.grey[700]
     },
     label: {
-      display: 'inline',
+      margin: '0',
+      padding: '0',
       fontWeight: theme.typography.fontWeightRegular,
       '&::after': {
         content: '":\x20"'
       }
     },
     link: {
-      fontWeight: theme.typography.fontWeightBold
+      fontWeight: theme.typography.fontWeightBold,
+      margin: '0 2px',
```

Comment:
Spacing of buttons should be controlled on the parent using `gap` property. Value should be in rem's calculated using theme helpers, `gap: theme.typography.pxToRem(theme.spacing(1))`.

Reply:
Fixed!